### PR TITLE
feat(Dashy):  Add support for "set" HIPIE inputs

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

**Comment by GordonSmith** [comment]
@jbrundage please review

**Comment by GordonSmith** [comment]
@drealeed FYI

**Comment by GordonSmith** [comment]
@jbrundage bump

File: packages/marshaller/src/ddl2/activities/roxie.ts [hunk 1]
```diff
@@ -22,25 +22,40 @@ function parseUrl(_: string): { url: string, querySet: string, queryID: string }
...
 
     @publish("", "string", "ESP Url (http://x.x.x.x:8002)")
     url: publish<this, string>;
     @publish("", "string", "Query Set")
     querySet: publish<this, string>;
     @publish("", "string", "Query ID")
     queryID: publish<this, string>;
+    @publish(false, "boolean", "Ignore provided DDL request")
+    ignoreHipieRequest: publish<this, boolean>;
+    @publish(false, "boolean", "Ignore provided DDL responze")
```
Comment (woodbr01) [inline packages/marshaller/src/ddl2/activities/roxie.ts]: typo in responze

Comment (GordonSmith) [reply packages/marshaller/src/ddl2/activities/roxie.ts]: Thx - resolved.

**Pull request merged**